### PR TITLE
V.2.0.0

### PR DESCRIPTION
**Pull request by mischah**
**Cha, Cha, Cha, Changes:**

+ ES6ify generator code
+ Install dependencies via Yarn 🎉
+ Adapt breaking changes of yeoman-generator 1.0.0 🎉
+ Add yarn lock file for the generator
+ Update dependencies of generator
+ Update http-fake-backend to v3.2.2

### CODE REVIEW

File: generators/app/index.js [hunk 1]
```diff
@@ -31,121 +32,120 @@ module.exports = yeoman.Base.extend({
...
+    this.fs.copy(
+      this.templatePath('test'),
+      this.destinationPath('test')
+    );
+  }
 
-  install: function () {
-    var that = this;
+  install() {
+    const that = this;
```
Comment (mischah) [inline generators/app/index.js]: Is this still the way to handle `this` in the days of ES6?

**Review by mischah — COMMENTED** [review]
@revrng  @krnlde Feedback appreciated 😘

File: generators/app/index.js (hunk 1 above)
Comment (krnlde) [reply generators/app/index.js]: Nope. You should use arrow functions wherever you need `this` inside the `install()` method.

So refactor `callback(error)` to `callback: (error) =>` and you are good to go.

File: generators/endpoint/index.js [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
+const helper = require('./promptingHelpers');
 
-module.exports = yeoman.Base.extend({
+module.exports = class extends Generator {
 
-  prompting: function () {
-    var that = this;
-    var done = that.async();
+  prompting() {
+    const that = this;
```
Comment (krnlde) [inline generators/endpoint/index.js]: Same here, see comment on https://github.com/micromata/generator-http-fake-backend/pull/2/files#diff-def8dd7c5bf7421ace8ec454d89f771cR131

File: generators/endpoint/index.js [hunk 3]
```diff
@@ -125,10 +125,10 @@ module.exports = yeoman.Base.extend({
         }
       });
     }
-  },
+  }
 
-  writing: function () {
-    var that = this;
+  writing() {
+    const that = this;
```
Comment (krnlde) [inline generators/endpoint/index.js]: same here, see comments on https://github.com/micromata/generator-http-fake-backend/pull/2/files#diff-def8dd7c5bf7421ace8ec454d89f771cR131

**Comment by mischah** [comment]
Nice. Thanks @krnlde 😘

**Pull request merged**